server/controllers/gym/getGym.ts

```diff
@@ -0,0 +1,34 @@
+import { NextFunction, Response, Request } from 'express';
+import { Gym, Image, Review, User } from '../../database/models';
+
+export default async function getGym(req: Request, res: Response, next: NextFunction) {
+  try {
+    const { id } = req.params;
```

Comment:
validate params to be a number using `hapi/joi` before running database find query